server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -245,6 +272,54 @@ public Entry withShards(ImmutableOpenMap<ShardId, ShardSnapshotStatus> shards) {
             );
```

Comment:
`withShards` was dropped in https://github.com/elastic/elasticsearch/commit/e91936512aa464593fa745f8234d48e6a54d50ca.
Seems like we can drop it as well, no usages in this branch.

Reply:
That's unrelated to the patches here?

Reply:
I think it's related to the third commit - at least this method was [removed](https://github.com/elastic/elasticsearch/commit/e91936512aa464593fa745f8234d48e6a54d50ca#diff-73ee7c916310b19deb06e97d8f66e7da737f1dff22c93bfced5b1e7fa071855dL211) in the original patch being ported

Reply:
Removed it